### PR TITLE
Reciprocal

### LINKED ISSUE #17736: reciprocal
implementing the reciprocal for paddle frontend

### PR DESCRIPTION
**Pull request by samthakur587**
Close #17736

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -276,3 +276,7 @@ def greater_than(self, y, name=None):
     @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
     def rsqrt(self, name=None):
         return ivy.reciprocal(ivy.sqrt(self._ivy_array))
+
+    @with_unsupported_dtypes({"2.5.0 and below": ("float16", "bfloat16")}, "paddle")
```
Comment (zaeemansari70) [inline ivy/functional/frontends/paddle/tensor/tensor.py:280]: This is failing for "int16" as it is not supported by paddle, you will probably come across more unsupported dtypes failures after adding this

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hi @samthakur587,

I have requested some changes below, kindly try fixing the errors and request a review from me when the tests are passing!

Feel free to ask any questions.
Thanks! :slightly_smiling_face:

File: ivy/functional/frontends/paddle/tensor/tensor.py (hunk 1 above)
Comment (samthakur587) [reply ivy/functional/frontends/paddle/tensor/tensor.py:280]: image

Hey @zaeemansari70 i am getting this error while testing the method. i am using pytest in vscode (wsl2-ubuntu-20.04). is there any different command to run test in cli.

thank you

**Comment by samthakur587** [comment]
hey @zaeemansari70  i changed the data type. now it,s passing all tests in my local.

 now its ready to merge. but if any changes required please let me know.

thank you

**Review by zaeemansari70 — APPROVED** [review]
The PR looks good to be merged :+1: 

Kindly resolve the conflicts so I can merge your PR :slightly_smiling_face:

**Comment by samthakur587** [comment]
hey @zaeemansari70  solved the merge conflict.

Now it's Ready to merge.

thank you

**Pull request merged**